shared/github_oauth.go

```diff
@@ -30,15 +31,22 @@ type User struct {
 
 // GitHubAccessControl encapsulates implementation details of access control for the wpt-metadata repository.
 type GitHubAccessControl interface {
-	IsValidAccessToken() (int, error)
-	IsValidWPTMember() (int, error)
+	// IsValid* functions also verify the access token with GitHub.
```

Comment:
```suggestion
	// IsValidWPTMember also verifies the access token with GitHub.
```

Reply:
Sorry I forgot to mention that there's a second `IsValidAdmin` coming in the next PR